### DISCUSSION TITLE
Fetch version from the contents of a release note

### DISCUSSION THREAD

**Discussion by huguesgr** [renovatebot/renovate#26889] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

latest

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### What is your question?

Hi! I'd like to fetch a package version by reading the contents of a release note. For example, [this project](https://github.com/tigera/operator/releases) usually mentions "Calico version: v3.27.0", and I would like to be able to catch that specific version.
Is this possible?

**Comment by rarkins** [comment]
Only if you can get it to work with customDatasources / customManagers
